tasks/MatchedCatalogsAnalysis.py

```diff
@@ -43,3 +42,36 @@ def __init__(self, config, *args, **kwargs):
 
     def run(self, matchedCatalog):
         return self.measure.run(matchedCatalog, self.config.connections.metric)
+
+
+class MatchedMultiCatalogAnalysisTaskConnections(MetricConnections,
```

Comment:
Isn't this identical to the `MatchedCatalogAnalysisTaskConnections` class?  We really should use inheritance to reduce duplication.

Reply:
They differ in their dimensions, so that it will match over all filters.

Reply:
Ah.  Of course.  I do think we could refactor to reduce some duplication, but I'm happy to leave that until the multi band match and visit level contexts are merged.